sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -221,6 +243,8 @@ public static Write write() {
 
       abstract Builder setTableId(String tableId);
 
+      abstract Builder setLocalhost(String host);
```

Comment:
Hi @dhalperi 

tbh I went with how same thing was done in DatastoreIO - and there setLocalhost() is also used - even though it is less needed there than in this IO... 

here if I remove withLocalhost() method then user (instead of invoking this one method) would have to:

1) specify SAME localhost value three times - three method invocations
2) set plaintext negotiation
3) set null credentials

And if any of those are forgotten then local testing will not work. My first impression is that as user of Beam I would like this to be as simple as possible?!